src/loadgenerator/step/locust-tasks/requirements.txt

```diff
@@ -0,0 +1,55 @@
+astroid==2.4.2
+cachetools==3.1.1
+certifi==2018.11.29
+chardet==3.0.4
+click==7.1.2
+ConfigArgParse==1.2.3
+Flask==1.1.2
+Flask-BasicAuth==0.2.0
+gevent==20.6.2
+geventhttpclient==1.4.4
+google-auth-httplib2==0.0.4
+google-cloud==0.34.0
+google-cloud-core==0.29.1
+google-cloud-monitoring==1.0.0
+google-cloud-monitoring-dashboards==1.0.0
+google-cloud-profiler==1.0.8
+google-cloud-trace==0.20.2
+googleapis-common-protos==1.52.0
```

Comment:
Do we need all these packages in order to send synthetic traffic with locust? Comment applies to entire file

Reply:
Yeah, I want to fix this... `pip-compile` couldn't find the locust version I needed and it looks like `pip freeze -r` listed way too many dependencies.